modules/installation-requirements-user-infra.adoc

```diff
@@ -167,7 +165,7 @@ ifdef::openshift-origin[]
 |Compute
 ifdef::ibm-z,ibm-power[|{op-system}]
 ifndef::ibm-z,ibm-power[|{op-system}]
-|2
+|2 GB
```

Comment:
Both vCPU terms edited in this PR should have the `GB` removed. The number is for the physical amount of [vCPUs](https://www.ibm.com/docs/en/license-metric-tool?topic=SS8JFY_9.2.0/com.ibm.lmt.doc/Inventory/overview/c_virtual_processor_core_licenses.html).